src/common/hooks/account/use-account-names.ts

```diff
@@ -0,0 +1,17 @@
+import { accountNameState } from '@store/accounts/names';
+import { useLoadable } from '@common/hooks/use-loadable';
+import { useCurrentAccount } from '@common/hooks/account/use-current-account';
+
+export function useAccountNames() {
+  return useLoadable(accountNameState);
+}
+
+export function useAccountDisplayName(index?: number) {
```

Comment:
Should each exported hook be in its own file of the same name if used throughout the app?

Reply:
I go back and forth on this, sometimes it's easier and less messy to include all "state" related hooks in a single file, rather than _many_ files with just a single wrapper hook exported